string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,22 @@
+/* exported isVowel */
+function isVowel(character) {
+  switch (character) {
+    case 'a':
+    case 'A':
+      return true;
+    case 'e':
+    case 'E':
+      return true;
+    case 'i':
+    case 'I':
+      return true;
+    case 'o':
+    case 'O':
+      return true;
+    case 'u':
+    case 'U':
+      return true;
+    default:
+      return false;
+  }
+}
```

Comment:
Nice job using a `switch` This is a great use case for one! We can actually simplify this even more, by converting everything to lowercase & only checking for lowercase vowels. Look at the snippet below.
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```

Reply:
I can't believe I didn't think of that! Thanks for the tip, will definitely do that next time there's a similar question.